graviti/manager/branch.py

```diff
@@ -37,7 +37,7 @@ class Branch(NamedCommit):
 
     """
 
-    parent_commit_id: str = _attr()
+    parent_commit_id: Optional[str] = _attr()
     title: str = _attr()
     description: str = _attr()
     committer: str = _attr()
```

Comment:
So we do not need to add if-else to create branch and other methods in this pr?

Reply:
Yes. Those will be updated in another pr.